### PR TITLE
Assignment 2b

### PR DESCRIPTION
**Pull request by belenfh999**

### CODE REVIEW

File: Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+	"@context":{
+		"vocab":"http://temperaturesystems.com/Products/#",
+		"Class01":"vocab:Class01",
+		"Sensor029":"Class01:Sensor029",
+		"Computer101":"Class01:Computer101",
+		"Measurement8401":"Sensor029:Measurement8401",
```
Comment (paoespinozarias) [inline Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld:7]: The definitions are a bit confusing. But don't worry, we'll see how to better define them in our ontology session :-)

File: Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	"@context":{
+		"vocab":"http://temperaturesystems.com/Products/#",
+		"Class01":"vocab:Class01",
+		"Sensor029":"Class01:Sensor029",
+		"Computer101":"Class01:Computer101",
+		"Measurement8401":"Sensor029:Measurement8401",
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"User10A":"Computer101:User10A"
+	},
+	"Class01":{
```
Comment (paoespinozarias) [inline Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld:11]: You don't need to define it.

File: Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+		"Sensor029":"Class01:Sensor029",
+		"Computer101":"Class01:Computer101",
+		"Measurement8401":"Sensor029:Measurement8401",
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"User10A":"Computer101:User10A"
+	},
+	"Class01":{					   
+		"@id":"http://temperaturesystems.com/Products/Class01",
+		"@type":"vocab:Class01",
+      		"Sensor029": {
```
Comment (paoespinozarias) [inline Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld:14]: The properties "includes", "hasMeasurement", and "hasOwner" are missing.

File: Assignment2/belenfh999-fromJSONtoJSON-LD.jsonld [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+		"age":"vocab:age",
+		"almaMater":"vocab:almaMater",
+		"parentsName":"vocab:parentsName",
+		"subjects":"vocab:subjects",
+		"name":"subjects:name",
+		"teachers":"subjects:teachers",
+		"description":"subjects:description"
+		
+	},
+	"@id":" http://example.com/semweb/people/",
```
Comment (paoespinozarias) [inline Assignment2/belenfh999-fromJSONtoJSON-LD.jsonld:15]: Ther is a blank space between: " h

File: Assignment2/belenfh999-fromJSONtoJSON-LD.jsonld [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+		"vocab":"http://example.com/semweb/",
+		"fullName":"vocab:fullName",
+		"hometown":"vocab:hometown",
+		"age":"vocab:age",
+		"almaMater":"vocab:almaMater",
+		"parentsName":"vocab:parentsName",
+		"subjects":"vocab:subjects",
+		"name":"subjects:name",
+		"teachers":"subjects:teachers",
+		"description":"subjects:description"
```
Comment (paoespinozarias) [inline Assignment2/belenfh999-fromJSONtoJSON-LD.jsonld:12]: "subjects" prefix is not defined. Define it or reuse already defined "vocab" instead.

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

File: Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld (hunk 1 above)
Comment (belenfh999) [reply Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld]: Sorry,  I did my best but I still find it a bit confusing

File: Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld (hunk 2 above)
Comment (belenfh999) [reply Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld]: You mean the xsd prefix, right? That's what i modified!

File: Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld (hunk 3 above)
Comment (belenfh999) [reply Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld]: I hope i fixed it with this new version!

File: Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld (hunk 2 above)
Comment (paoespinozarias) [reply Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld]: No, I meant delete "Class01":{ and also the closing "}".
Please, restore the xsd prefix definition.

File: Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld (hunk 3 above)
Comment (paoespinozarias) [reply Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld]: Now, it is ok!

File: Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld (hunk 1 above)
Comment (paoespinozarias) [reply Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld]: Not worries ;-)

File: Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld [hunk 6]
```diff
@@ -1,43 +1,62 @@
...
+    "Computer101": "Class01:Computer101",
+    "Measurement8401": "Sensor029:Measurement8401",
+    "User10A": "Computer101:User10A",
+    "includes": "properties:includes",
+    "hasMeasurement": "properties:hasMeasurement",
+    "temperature": "properties:temperature",
+    "atTime": "properties:atTime",
+    "hasOwner": "properties:hasOwner",
+    "name": "properties:name"
+  },
```
Comment (paoespinozarias) [inline Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld:16]: Please, restore the xsd prefix definition.

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

File: Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld (hunk 6 above)
Comment (belenfh999) [reply Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld:17]: Now it is back, sorry for misunderstanding  your request

**Review by paoespinozarias — APPROVED** [review]

**Pull request merged**